plugin/src/main/resources/scalac-plugin.xml

```diff
@@ -0,0 +1,4 @@
+<plugin>
+  <name>dependency-analyzer</name>
+  <classname>plugin.src.main.scala.io.github.retronym.dependencyanalyzer.DependencyAnalyzer</classname>
```

Comment:
It's a scala compiler plugin descriptor that contains the entry point for the plugin.
See http://www.scala-lang.org/old/node/140